docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,169 @@ eclagent:
+    used by the HPCC Systems platform. The credentials for that HPCC Systems
+    administrators account therefore must be secured either using Kubernetes
+    secrets or the Hashicorp Vault. </para>
+
+    <sect2 id="Configuring_For_KubernetesSecrets">
+      <title>Securing Credentials in Kubernetes</title>
+
+      <para>To create a secret in Kubernetes for the HPCC Administrators user
+      account from a command line interface to Kubernetes, execute a command
+      similar to the following example. Note the secret name, is
+      "admincredssecretname" in this example. The HPCC Administrators user
+      account "username" and "password" key/values are required, and
+      additional properties are ignored.</para>
+
+      <programlisting lang="bash">kubectl create secret generic admincredssecretname --from-literal=username=hpcc_admin \
+  --from-literal=password=t0pS3cr3tP@ssw0rd
+</programlisting>
+
+      <para>This sets the values for the Administrator's username and password
+      in Kubernetes and sets them as secrets. At this point you can verify the
+      secret you just created by executing the following command on the
+      Kubernetes command line interface.</para>
+
+      <programlisting lang="bash">kubectl get secret admincredssecretname</programlisting>
+
+      <para>For more information about Kubernetes see the appropriate
+      Kubernetes documentation for your implementation.</para>
+
+      <sect3 id="Deploying_TheKubernetesSecret">
+        <title>Deploying the Kubernetes Secret</title>
+
+        <para>To deploy the Kubernetes secrets override the "secrets:" section
+        in HPCC-Platform/helm/hpcc/values.yaml, or deploy with your own
+        customized chart. For more information about customizing your HPCC
+        Systems containerized deployment see the above sections on
+        customization techniques.</para>
+
+        <para>In your chart, create a unique key name used to reference the
+        secret, and set it to the secret value that you created in the
+        previous step. In the above example it was
+        "admincredssecretname".</para>
+
+        <para>You can optionally define an additional alternate name, such as,
+        "admincredsaltmountname" which can be used with another Active
+        Directory server. The alternate name is illustrated in the example
+        below and should be added to your chart if necessary.</para>
+
+        <para>The "admincredsmountname" key/value pair already exists by
+        default in the HPCC Systems delivered values.yaml file. The key is
+        referenced in the component's ldap.yaml file. You may override these
+        and add additional key/values as needed.</para>
+
+        <programlisting lang="bash">  secrets:
+     authn:
+       admincredsmountname: "admincredssecretname"       #exernalize HPCC Admin creds
+       admincredsaltmountname: "admincredsaltsecretname" #exernalize alternate HPCC Admin creds
```

Comment:
I am trying to get a little more information about the alt secret. I don't think the platform uses it at all. I think it's only a way to provide a secret to a custom component that is looking for it. I will follow up with more information when I have it. IMO, it may be something we can remove. Or, have a general statement that if you have other components that may need credentials or secrets, you are free to create them as needed.